.changeset/friendly-countries-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/project-access': patch
```

Comment:
Hi @Jimmy-Joseph19,

I'd say because we add functionality this addition could be considered as minor update.

>From https://semver.org/:
>MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes

Reply:
i changed to minor.